lab-shaun/lib/greet.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+let greet = module.exports = {}
+
+greet.hello = function(name){
```

Comment:
Part of the technical requirements was to validate that the input was a string (not a number, Boolean, etc.)